lib/config.js

```diff
@@ -0,0 +1,30 @@
+/*!
+ * Copyright (c) 2021-2022 Digital Bazaar, Inc. All rights reserved.
+ */
+import {join, dirname} from 'path';
+
+// es6 imports and exports don't support __dirname
+// so we need to use process command working dir.
+const homeDir = () => dirname(process.cwd());
```

Comment:
`__dirname` and `process.cwd()` are not the same thing.  These paths look to be relative to the source file, so you probably want the direct [`__dirname` equiavlent](https://gist.github.com/sindresorhus/a39789f98801d908bbc7ff3ecc99d99c#what-do-i-use-instead-of-__dirname-and-__filename):

```js
import path from 'node:path';

const __dirname = path.dirname(fileURLToPath(import.meta.url));
```

"homeDir" is also a confusing name since it can imply the users actual home dir.

Reply:
Yeah I caught that and started using `import.meta.url` here: https://github.com/digitalbazaar/mocha-w3c-interop-reporter/pull/10/commits/f0a67ef24a1cc090e952d55422bb8ce8fccdef35

I think maybe `packageRootDir` would be a better name?

Reply:
use `app-root-path` package: https://github.com/digitalbazaar/mocha-w3c-interop-reporter/pull/10/commits/f1fb1797df0d5955ba5de1f51f68d98ee4f0efed